Sources/xcodeproj/Objects/Project/PBXProject.swift

```diff
@@ -108,7 +108,31 @@ public final class PBXProject: PBXObject {
     }
 
     /// Project attributes.
-    public var attributes: [String: Any]
+    var attributeReferences: [PBXObjectReference: [String: Any]]
```

Comment:
I opted for making this one internal instead of using `PBXTarget` as a key. That would have required making `PBXTarget` conform the `Hashable` protocol.